library/ctr_drbg.c

```diff
@@ -94,11 +94,15 @@ int mbedtls_ctr_drbg_seed_entropy_len(
     /*
```

Comment:
I don't know if you realise it, but you've changed the access permissions on the file. Please revert.

Reply:
Sorry, I don't know how it happened.
I've fixed it.
Thanks!